python/hyperspace/indexconfig.py

```diff
@@ -0,0 +1,14 @@
+class IndexConfig:
```

Comment:
In Scala, `IndexConfig` is in the `com.microsoft.hyperspace.index` namespace.  I'm wondering if you should try to match this in Python.

Reply:
This was deliberately removed this in python. Initially original namespace was com.microsoft...delta/pyspark follows similar shorter namespace for python APIs to allow import with minimal code.
@imback82  mentioned to follow similar style.